src/core/lib/security/transport/security_connector.h

```diff
@@ -154,11 +144,10 @@ void grpc_channel_security_connector_check_call_host(
     const char *host, grpc_auth_context *auth_context,
     grpc_security_call_host_check_cb cb, void *user_data);
 
-/* Handshake. */
-void grpc_channel_security_connector_do_handshake(
+/* Registers handshakers with \a handshake_mgr. */
+void grpc_channel_security_connector_create_handshakers(
```

Comment:
That seems a bit long.  I think I prefer the current name, but as I mentioned above, please feel free to push back if you feel strongly.